Jenkinsfile

```diff
@@ -0,0 +1,10 @@
+library("tdr-jenkinslib")
+
+terraformDeployJob(
+  //Always default to the management (mgmt) environment as Grafana should only be deployed in this environment
+  stage: "mgmt",
```

Comment:
So we'll revisit this in [TDR-540](https://national-archives.atlassian.net/browse/TDR-540) to remove the stage parameters from this Jenkins job?

Reply:
Hopefully yes, we'll be able to get rid of this.